### DISCUSSION TITLE
What is the api url for self hosted?

### DISCUSSION THREAD

**Discussion by berts83231** [calcom/docker#243] [General]
We are using Docker to self host an instance of Cal, under the "Platform" license.

We want to use the api to automate some more complex processes.
The documentation does not seem to provide the answers.

1) What is the url endpoint for the api?

2) What env vars do I need to configure in Docker?

3) Is this the api key needed to access our api endpoint?
https://nimb.ws/lHyMXa

Thanks

**Comment by zomars** [comment]
Have you [requested access to the private API repo](https://cal.com/docs/introduction/quick-start/self-hosting/installation/platform/api-submodule)?

**Comment by berts83231** [comment]
Hi Omar,

I have not setup the API part of the app.

I'm not building a custom docker image from the repo. So I don't think those instructions apply 

I'm pulling the docker image straight of off Dockerhub.

**Reply by zomars** [reply]
I think I'm not understanding what are you trying to achieve. Are you trying to self-host your own instance but don't want to self-host the API as well?

**Reply by berts83231** [reply]
I do want to use the Cal api on our self hosted app.
I'm using the Docker image without modification from  https://hub.docker.com/r/calcom/cal.com

- Is the api server running? (I guess it is not)
- How do you setup the domain the api will use?
- What is the endpoint url?
- is this how we setup api keys for the self hosted version? https://nimb.ws/lHyMXa

**Reply by Matheus-h-Campos** [reply]
@berts83231 Have you found a solution to that?

**Comment by asadriko** [comment]
The app is awesome, but when you actually need some help or practical tips, it feels like everyone just goes silent 😅. Makes me wonder if it’s a lack of active users, documentation, or if people just don’t know much about self-hosting.

Honestly, I think the best way forward is for anyone who figures something out to come back here and share it, even if it’s just a basic step-by-step. That’s how the community really grows! If you’ve solved any issues, please post here, even if it seems simple — it makes a difference for people just starting out.

Anyone else struggling with this? Let’s share some experiences!
